apps/changelog-generator/src/Cli.ts

```diff
@@ -1,8 +1,108 @@
-import * as Command from '@effect/cli/Command';
+import { Command, Options } from '@effect/cli';
+import type { ConfigType, DeploymentConfigV2 } from '@blocksense/config-types';
+import {
+  configDir,
+  readConfig,
+  readEvmDeployment,
+} from '@blocksense/config-types';
+import { Console, Effect } from 'effect';
+import Table from 'cli-table3';
+import type { NetworkName } from '@blocksense/base-utils';
+import { parseNetworkName } from '@blocksense/base-utils';
+import { Terminal } from '@effect/platform';
 
-const command = Command.make('hello');
+const listFeeds = Command.make(
+  'list-feeds',
+  {
+    dir: Options.directory('dir').pipe(Options.withDefault(configDir)),
+  },
+  ({ dir }) =>
+    Effect.gen(function* (_) {
```

Comment:
I just read the Effect TS docs about this `_` parameter. TL;DR you don't need it, as we're using TS 5.8+. 
https://effect.website/docs/getting-started/using-generators/#adapter